modules/SetupWizard/Android.mk

```diff
@@ -9,5 +9,6 @@ else
 endif
 
 LOCAL_PRIVILEGED_MODULE := true
+LOCAL_OVERRIDES_PACKAGES := Provision
```

Comment:
The Provision app is just a minimal setup wizard. It just sets a few
properties that the Google SetupWizard also sets. So it's safe to remove on
all versions of Android.

See:
https://android.googlesource.com/platform/packages/apps/Provision/+/356bff898c6f33334d20ea68bd5c49f7a2ef9b6b/src/com/android/provision/DefaultActivity.java

On 31 Aug 2016 18:17, "Maarten Fonville" <notifications@github.com> wrote:

> In modules/SetupWizard/Android.mk
> <https://github.com/opengapps/aosp_build/pull/76#discussion_r77019461>:
>
> > @@ -9,5 +9,6 @@ else
> >  endif
> >
> >  LOCAL_PRIVILEGED_MODULE := true
> > +LOCAL_OVERRIDES_PACKAGES := Provision
>
> On Marshmallow there was no conflict?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opengapps/aosp_build/pull/76/files/8d595354580950f8b18e98b544041d58e152248c..247b5f5a45b0387e3a91800a78698694e1346d6f#r77019461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATzmuegRHwGfZElAn8IAd-6PdI-HGinks5qlaj7gaJpZM4JxqBZ>
> .
>